config-paths.js

```diff
@@ -0,0 +1,23 @@
+const { app } = require('electron');
+const { homedir } = require('os');
+const { resolve } = require('path');
+const { existsSync } = require('fs');
+
+const home = homedir();
+const root = resolve(app.getPath('appData'), 'HyperTerm');
```

Comment:
Interestingly, I tried that in dev and the application was always reporting itself as `Electron` - despite the `name` and `productName` fields in `package.json`. This was my work around, albeit not as elegant.

Is this a problem with my environment? My experience with Electron is thin.